Cabal/src/Distribution/Verbosity.hs

```diff
@@ -217,6 +218,7 @@ parsecVerbosity = parseIntVerbosity <|> parseStringVerbosity
             "stderr"     -> return verboseStderr
             "stdout"     -> return verboseNoStderr
             "nowarn"     -> return verboseNoWarn
+            "nowarnexperimental" -> return verboseNoWarnExperimental
```

Comment:
`"nowarnexperimental"` is a bit on the lengthy side, if anyone have suggestions, I'd be much obliged.

Reply:
I'm fine with length. However, the lack of separation of words make it hard for me to read. I think cabal uses hyphen in it's commandline? Could we somehow adopt the GHC convention perhaps (not sure how): https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/using-warnings.html

Reply:
We could, but it would look really weird, and perhaps even out of place, in our context.

Reply:
Perhaps somebody on IRC would have an idea how to make `--nowarnexperimental` more readable?

Reply:
BTW, I can't find the other options from this code snippet in `cabal help`, so I lack context. How do I see them?

Reply:
They're not in the `cabal help`, see here:

https://github.com/haskell/cabal/blob/724e9f5cb458909766bb31e62262803a63a307fd/Cabal/src/Distribution/Verbosity.hs#L242-L249

and

https://github.com/haskell/cabal/blob/724e9f5cb458909766bb31e62262803a63a307fd/Cabal/src/Distribution/Verbosity/Internal.hs#L17-L25

Reply:
Thank you. All the others have the same style, so IMHO `+nowarnexperimental` is perfectly fine if we don't feel like changing everything (and worrying about backward compatibility).